app/components/post_draft/draft_input/draft_input.js

```diff
@@ -434,6 +434,7 @@ export default class DraftInput extends PureComponent {
                         disableScrollViewPanResponder={true}
                     >
                         <PostInput
+                            testID={testID + '.post.input'}
```

Comment:
Are we able to use string interpolation? ex. \`${testID}post.input\` and in other places in this file.

Reply:
@hahmadia Haha, I was wondering whether to do that or not but I hesitated because of your comment in the previous PR. But I just updated all of it to use string interpolation.